src/rules.js

```diff
@@ -43,10 +43,10 @@ block.def = edit(block.def)
   .getRegex();
 
 block.bullet = /(?:[*+-]|\d{1,9}[.)])/;
-block.item = /^( *)(bull) ?[^\n]*(?:\n(?! *bull ?)[^\n]*)*/;
-block.item = edit(block.item, 'gm')
-  .replace(/bull/g, block.bullet)
-  .getRegex();
+// block.item = /^( {0,3})(bull)( [^\n]*(?:\n(?! {0,3}bull )[^\n]*)*\n?)/;
+// block.item = edit(block.item, 'gm')
+//   .replace(/bull/g, block.bullet)
+//   .getRegex();
```

Comment:
this should actually be deleted if it is not used

Reply:
Done.

Reply:
Done.